Introduction, DML need and trade-offs

Kloppie5 opened the pull request:
Wrote small, extendable texts for introduction, the need for distributed machine learning and the trade-offs in the results and conclusion, possible to scale with the rest of the document dependent on the amount of text that is going to be removed from the machine learning enumeration

Kloppie5 committed:
Wrote small introduction, to be expanded

Kloppie5 committed:
The need for Distributed Machine Learning

Kloppie5 committed:
Trade-off in results and conclusion

jverbraeken committed:
Merge branch 'master' into introduction

mwolting commented on introduction.tex:
```diff
@@ -1,8 +1,18 @@
+
+Most Machine Learning algorithms assume that all data can be easily accessed. This is not always the case, as some forms of data are inherently distributed, like transaction information for multinationals, or otherwise too big to store on single machines, like astronomical data. Regular Machine Learning algorithms have to be changed to accommodate for these circumstances to still be able to extra ...
+
+In this paper, we will give an overview of the current state-of-the-art by going over the different theories and technologies used for Distributed Machine Learning and some of the biggest implementations.
```
Could be a good idea to concretely mention our research questions here, and refer to the appendix that motivates them. Would set up context for the rest of the paper.

mwolting replied:
Could also be more formal.

Kloppie5 replied:
I think that explicitely mentioning the questions is not in style with an introduction

mwolting replied:
They don't have to be straight up bullet points, but I'd say this paragraph should mention them so that the intro is symmetrical with the conclusion

mwolting commented on introduction.tex:
```diff
@@ -1,8 +1,18 @@
+% Very well-structured document. General presentation of the content (text and figures) is very effective. 
+% Excellent expressed and formulated report. Document has a smooth flow. Spelling and grammatically error free.
+
+With the ever increasing amount of data created every day and the prevalence of Machine Learning (ML) algorithms in analyzing all this data, a lot of research is being done on the application of Distributed Machine Learning approaches. Because of the many different approaches to dealing with so-called Big Data, we would like to give an overview of the current state-of-the-art of Distributed Machi ...
```
Could be more formal ("With" -> "Due to", "is being done" -> "is occurring", "Because of" -> "To ", "we would like to give" -> "we provide") + final sentence could be clearer on how big data relates to DML.

mwolting commented on introduction.tex:
```diff
@@ -1,8 +1,18 @@
+
+With the ever increasing amount of data created every day and the prevalence of Machine Learning (ML) algorithms in analyzing all this data, a lot of research is being done on the application of Distributed Machine Learning approaches. Because of the many different approaches to dealing with so-called Big Data, we would like to give an overview of the current state-of-the-art of Distributed Machi ...
+
+Most Machine Learning algorithms assume that all data can be easily accessed. This is not always the case, as some forms of data are inherently distributed, like transaction information for multinationals, or otherwise too big to store on single machines, like astronomical data. Regular Machine Learning algorithms have to be changed to accommodate for these circumstances to still be able to extra ...
```
First sentence could be split into about three shorter ones. Final sentence could be less structurally repetitive.

mwolting reviewed (changes requested):
I'm fine with merging despite vocab & sentence structure stuff, we can do that for the whole document once the content is there. I do think the intro should be changed to include research questions before merge.

Kloppie5 committed:
Improved Introduction

Made the Introduction more formal and reduced the length of some of the sentences

jverbraeken committed:
Merge branch 'master' into introduction

jverbraeken reviewed (changes requested):
We have to state our research question and sub-research questions. The most logical place to do this is in the introduction. I would also refer to the appendix where the motivation of the research questions can be found and to the conclusion where a concise answer to the research questions can be found.
Lastly, we should also provide a high-level description of the structure of the paper. This can be done whilst explaining the research questions

jverbraeken committed:
Merge branch 'master' into introduction

Kloppie5 committed:
Atmp 3

Kloppie5 committed:
Atmp 4

jverbraeken committed:
Minor change

jverbraeken committed:
Merge branch 'master' into introduction
